### ISSUE TITLE
Wrong get_radius function

### ISSUE THREAD

**Issue by fsk119** [lukasruff/Deep-SVDD-PyTorch#3]
https://github.com/lukasruff/Deep-SVDD-PyTorch/blob/1919546eda13b6b70e1701bde08dedf13b92bc11/src/optim/deepSVDD_trainer.py#L179-L181

Hi, I think this line 181 is wrong. We can modify this line to 
```
np.quantile(np.sqrt(dist.clone().data.cpu().numpy()), 1-nu). 
```
Because dist is (feat - center)**2 and we are comparing dist to R^2 in https://github.com/lukasruff/Deep-SVDD-PyTorch/blob/1919546eda13b6b70e1701bde08dedf13b92bc11/src/optim/deepSVDD_trainer.py#L131.

**Comment by lukasruff** [comment]
Thanks for catching this error! 
You're correct and I've fixed line 181 in commit 77b5d33816cb454aa30264a0353401f6f6e8b59e according to your suggestion.

**Comment by jhl13** [comment]
> Thanks for catching this error!
> You're correct and I've fixed line 181 in commit [77b5d33](https://github.com/lukasruff/Deep-SVDD-PyTorch/commit/77b5d33816cb454aa30264a0353401f6f6e8b59e) according to your suggestion.

Hi, according to the paper, the dist should be sorted, but I don't find any code to do this.
